utils/utils.go

```diff
@@ -53,7 +55,41 @@ func TruncateString(str string, maxLength int) string {
 
 	return str[0:maxLength-3] + "..."
 }
+func RunCmd(name string, args ...string) (string, error) {
```

Comment:
Could this and `RunCmdAndWait` (below) be combined?

Reply:
Probably. I did not want to change `RunCmdAndWait` since it was already used in a bunch of places and I wanted to avoid refactoring

Reply:
I would combine RunCmd to RunCmdAndWait. the cost of refactoring will be small because we can just do like below.

```
err := utils.RunCmdAndWait("kubectl", "apply", "-f", daprManifestPath) 
```
->
```
_, err := utils.RunCmdAndWait("kubectl", "apply", "-f", daprManifestPath) 
```